engine/src/main/java/org/hibernate/search/util/logging/impl/Log.java

```diff
@@ -1029,4 +1029,8 @@
 	@Message(id = 335, value = "Unable to build a Lucene query from the query string '%1$s'.")
 	EmptyQueryException unableToBuildLuceneQueryFromQueryString(String query);
 
+	@LogMessage(level = Level.DEBUG)
+	@Message(id = 336, value = "A file could not be deleted: likely lock contention. Not a problem for index replications as it will be attempted again in the future.")
+	void fileDeleFailureIgnored(@Cause IOException e);
```

Comment:
Shouldn't this be "fileDeleteFailureIgnored"? I don't get why we would use an abbreviation here.